src/contact.ts

```diff
@@ -277,52 +278,40 @@ export class Contact implements Sayable {
   /**
    * Contact gender
    *
-   * @returns Gender.Male(2) | Gender.Female(1) | Gender.Unknown(0)
```

Comment:
Just use `Gender.Male` `Gender.Female` and `Gender.Unknown`, there's no need to refer the number value of them because that's why ENUM type exists: Do not rely on the magic numbers.

So does the other ENUM types.